reference/configuration/assetic.rst

```diff
@@ -50,7 +50,7 @@ Full Default Configuration
                 some_filter:                 []
             workers:
                 # see https://github.com/symfony/AsseticBundle/pull/119
-                # Cache can also be busted via the framework.templating.assets_version
+                # Cache can also be busted via the framework.assets.version
```

Comment:
correct, but i see this happening more often, things are a bit cryptic for the people that is migrating to 2.7 since they have to be savvy in debugging things like this. I am cool either way, just pointing things.